.github/actions/deploy-hyp3/action.yml

```diff
@@ -56,6 +56,12 @@ inputs:
   REQUIRED_SURPLUS:
     description: "Amount by which month-to-date budget must exceed month-to-date spending to increase fleet size, in dollars. Ignored when DefaultMaxvCpus = ExpandedMaxvCpus."
     required: true
+  PERMISSIONS_BOUNDARY_ARN:
+    description: "Arn of permissions boundary to provide to any created IAM roles"
+    required: true
+  DEPLOY_LAMBDAS_IN_VPC:
+    description: "Whether or not to deploy lambda functions in a VPC"
+    required: false
```

Comment:
The stack expects the string `true` or `false`, so if we're not requiring this input we should at least default to the string `false`